dynamiq/connections/connections.py

```diff
@@ -795,9 +795,9 @@ def conn_params(self) -> dict:
                 -the url with the key 'url'.
         """
         return {
-            "apikey": self.api_key,
+            "api_key": self.api_key,
             "project_id": self.project_id,
-            "url": self.url,
+            "api_base": self.url,
```

Comment:
Why we change `url` to `api_base`? By doc it seems `url` is correct https://docs.litellm.ai/docs/providers/watsonx#passing-credentials-as-parameters